internal/jobqueue/fsjobqueue/fsjobqueue.go

```diff
@@ -59,6 +59,10 @@ type job struct {
 	Canceled bool `json:"canceled,omitempty"`
 }
 
+// The size of channels used in fsJobQueue for queueing jobs.
+// Note that each job type has its own queue.
+const channelSize = 100
```

Comment:
What happens when we reach this limit. Maybe that's a question for @larskarlitski ;)

Reply:
Bad things. (Enqueuing will hang forever.)

This has always been the case. Ondřej just moved it to a constant.

Reply:
Yeah, this is nothing to address in this PR, but we probably need to. Or at least bump it by a factor of ten or something?